pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -55,6 +55,37 @@ def load(self, f):
         except MissingSectionHeaderError:
             raise TypeError(_("Treeinfo file should have INI format"))
 
+    def sections_from_general(self, parser):
```

Comment:
general is formed by data from other sections:
https://productmd.readthedocs.io/en/latest/treeinfo-1.0.html
for this case, it had data on general, but not on the "original sections", so I built these sections from the general, using the link above as reference

Reply:
@fabricio-aguiar , @daviddavis , does it mean that during publication we'll get a different treeinfo file?
Or do we keep treeinfo file untouched?

Reply:
treeinfo file will be untouched, it just changes the way we read it